apiary.apib

```diff
@@ -798,8 +808,8 @@ form of choices.
                     }
                 ]
             }
-            
-## Answer [/customers/{cid}/surveys/{sid}/questions/{qid}/answer]
+
+## Answer [/customers/{cid}/answers]
```

Comment:
let's discuss this one. I never need the full list of answers. What I need is the answer to a given question of a given survey.